config/locales/chapters.yml

```diff
@@ -0,0 +1,86 @@
+          description: We were very proud to bring yet another BP chapter in a non-tech spot ✨
+          url: "https://website.sdwight.workers.dev/"
+          image: "chapters/alabama.png"
+        - name: University of Colorado Boulder
+          description: We are currently in the process of establishing in CU Boulder 💕
+          url: ""
+          image: "chapters/colorado.png"
+
+      why-blueprint:
+        title: Why Blueprint? 
+        description: >
+
+        bridge: Bridge the gap between tech and non-profits. 
+        bridge_description: >
+          It’s true that non-profits lack technical resources to do their meaningful work. It’s also true that tech can’t solve global challenges. 
+          Therefore, we are creating collaborative spaces both for NPOs and aspiring technologists to make positive changes together! 
+        mark: Make a mark in your local community. 
+        mark_description: > 
+          Whether your university has a lot of clubs or not, you can be the one to bring accessible pathways to get into tech for a range of diverse voices. 
+          We want to further our DEI efforts with you! 
+        learn: Learn and grow. 
+        learn_description: > 
+          You and your peers can develop skills in software development, digital product design, event management, and so much more. 
+          And all of this for a good cause! 
+      chapter-timeline:
+        submission: >
+          Fill out the form at the bottom of this page and someone on our team will reach out to you to get the process started!
+          If you have any questions or concerns about anything in this timeline, please email
+          <a href="mailto:team@calblueprint.org">team@calblueprint.org</a>.
+        find_team: >
+          Reach out to friends and peers to form your executive team to have a starting team in building your chapter. 
+        school_policies: >
+          Ensure that you align your chapter formation with existing student organization policies.
+        onboard: >
+          Read over the provided Outreach & Awareness, Project Leader, Designer, and Developer guides to understand the structure and function of Blueprint. Don’t hesitate to reach out if you have any questions or need any advice! 
+        recruit_members: >
+          Create marketing material to spread the word in your community and find people to join your project teams. 
+        find_npos: >
+          Reach out to NPOs in your community and tell them about what your Blueprint chapter can offer. Learn about their mission, background, projects and technological needs to decide if they would be a good fit as an NPO Partner.
+      faq:
+        q1:
+          question: What kind of applications does Blueprint typically build?
+          answer: >
+            Blueprint typically builds Rails applications with Postgres databases and React on the front-end. 
+            Mobile applications are built natively for the desired platform. 
```

Comment:
For example, we don't build mobile applications natively all the time, and we've been moving away from Rails.